debian/entry.sh

```diff
@@ -72,9 +72,14 @@ function init_non_systemd()
 	# echo error message, when executable file doesn't exist.
 	if [ $? == '0' ]; then
 		shift
-		"$CMD" "$@" &
-		pid=$!
-		wait $pid
+		if [ ! -z "$RESIN" ] && [ ! -z "$RESIN_DEVICE_UUID" ]; then
```

Comment:
This means that an interactive command will always start a udev process. Can you isolate just the command running part in its own function and call that when we're not in a resin context?